modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach errors
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node.
+
+Mounting on a new node is not possible, however, because the failed node was unable to properly unmount the attached volume.
+
+A multi-attach error is reported, such as the one shown in the following example:
+
+[source,terminal]
+.Example output
```

Comment:
Not critical, but I'd say that this heading isn't quite necessary since you're introducing it with the previous sentence.